script/upload_to_aptly

```diff
@@ -12,31 +12,41 @@ set -exu
 # DOESN'T HAVE A SIGNING SUBKEY.
 #
 # Date          Channel       Fingerprint           Full Key
```

Comment:
Add `9BD198A2D848C482E3550697A8580BDC82D3DC6C` with the "NOT ACTIVE" date as is done in `upload_to_rpm_repo` below. Since we're using it in the case statement in this script, we should document it here.

Reply:
to be fixed in a future PR